sparql11/data-sparql11/cast/cast-bool.rq

```diff
@@ -0,0 +1,11 @@
+PREFIX : <http://example.org/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+SELECT
+	?a
+	?v
+	(xsd:boolean(?v) AS ?boolean)
+WHERE {
+    ?a :p ?v .
+}
+ORDER BY ?s

```

Comment:
Should that be "ORDER BY ?a"?

Reply:
Oops. The ordering was only for my benefit while working through the tests, and should just be removed.